lib/reline/ansi.rb

```diff
@@ -118,12 +128,6 @@ def self.set_default_key_bindings_comprehensive_list(config)
       # Del is 0x08
       # Arrow keys are the same of KDE
 
-      # iTerm2
-      [27, 27, 91, 67] => :em_next_word,    # Option+→, extended entry
-      [27, 27, 91, 68] => :ed_prev_word,    # Option+←, extended entry
-      [195, 166] => :em_next_word,          # Option+f
-      [195, 162] => :ed_prev_word,          # Option+b
```

Comment:
`[195, 166]` and `[195,162]` are unicode bytes for `æ` and `â`. It's not an escape sequence.

In mac, `option+f` `option+b` will insert `ƒ` `∫` in any textarea and also in iTerm2, but we don't want to add `'ƒ'.bytes` `'∫'.bytes` to this list. Same for `æ` and `â`.